utils/compare_dependency_constraints.py

```diff
@@ -93,8 +110,18 @@ def compare_constraints(images_contained_in_native: list[str]):
             )
         )
 
-        for discrepancy in discrepancies:
-            print(str(discrepancy))
+    for discrepancy in discrepancies:
+        file_pyproject_path = DOCKER_FOLDER / f"{discrepancy.image}/pyproject.toml"
+        if file_pyproject_path.exists():
+            file_path = file_pyproject_path
+        else:
+            file_path = DOCKER_FOLDER / f"{discrepancy.image}/Pipfile"
+        line_number = find_library_line_number(discrepancy.dependency, file_path)
+
+        print(
+            f"::error file={file_path},line={line_number},endLine={line_number},title=Native Image Discrepancy::{discrepancy}"
```

Comment:
Now when we're not in a CLI tool anymore, let's format the error message to something like 

> `requests` is set as `*` in `py3-tools image`, but `{'extras': ['socks'], 'version': '*'}` in the native image. This discrepancy may cause issues when running content.